app/views/items/index.html.erb

```diff
@@ -7,11 +7,13 @@
 
 <% @items.each do |item| %>
   <section id='item-<%= item.id%>'>
+    <% if item.active? %>
```

Comment:
Question for our check-in with Megan this afternoon:
- Does line 10 violate MVC? I'm not sure if we need to create a method in the Item model that checks for all active items.